stripe/src/test/java/com/stripe/android/paymentsheet/CurrencyFormatterTest.kt

```diff
@@ -20,14 +20,14 @@ class CurrencyFormatterTest {
 
```

Comment:
Is CurrencyFormatterTest and PaymentUtilsTest related to java 11 change?

Reply:
Yes, the underlying implementation of `Currency` changed in Java 11.